### DISCUSSION TITLE
React native Macos support

### DISCUSSION THREAD

**Discussion by schnAlex** [software-mansion/react-native-screens#1575] [Ideas]
As of some issues here and at react-navigation is seems that a specific Version of react native screens is working on macOS. But it is Not Listed as a Supported Platform. Are there and Plans on Fully Supporting the macOS platform? Would be glad to contribute. 
Cheers

**Comment by tboba** [comment]
Hi @schnAlex, I think that supporting MacOS may be very tricky here, as MacOS apps don't have a conteption of stack - probably we could need to simulate the behavior of iOS apps. There were some thoughts about it, but it's not our priority right now.

Also, if you would like to contribute for that, go ahead and please try 🙏 Everyone's help is more than welcome here! ❤️
